### ISSUE TITLE
Target [Illuminate\Contracts\Routing\ResponseFactory] is not instantiable on calling Response::json using Response Facade in Lumen 5.4.3

### ISSUE THREAD

**Issue by sumeeti** [laravel/lumen-framework#567]
- Laravel/Lumen Version: 5.4.3
- PHP Version:7.0.14
- Database Driver & Version:

### Description

Here's my code
```
       if (empty($errors))
        {
            /* Success case */
        }
        else
        {
            $response = array('success' => false, 'errors' => $errors);
        }

        return Response::json($response);
```

Here's the error
```
in Container.php line 804
at Container->notInstantiable('Illuminate\\Contracts\\Routing\\ResponseFactory') in Container.php line 687
at Container->build('Illuminate\\Contracts\\Routing\\ResponseFactory') in Container.php line 565
at Container->make('Illuminate\\Contracts\\Routing\\ResponseFactory') in Application.php line 208
at Application->make('Illuminate\\Contracts\\Routing\\ResponseFactory') in Container.php line 1070
at Container->offsetGet('Illuminate\\Contracts\\Routing\\ResponseFactory') in Facade.php line 159
at Facade::resolveFacadeInstance('Illuminate\\Contracts\\Routing\\ResponseFactory') in Facade.php line 128
at Facade::getFacadeRoot() in Facade.php line 215
at Facade::__callStatic('json', array(array(***))) in AppResponse.php line 31
```

### Steps To Reproduce

Use Response::json using Response Facade to return responses.

**Comment by ismaail** [comment]
Same error calling `response()` and `response()->json()`

the workaround is returning `new \Illuminate\Http\Response($response)` or `new \Illuminate\Http\JsonResponse($response)`

**Comment by ismaail** [comment]
Found a fix from [tymondesigns/jwt-auth#532](https://github.com/tymondesigns/jwt-auth/pull/532)

In **bootstrap/app.php** uncomment this line 
`$app->register(App\Providers\AppServiceProvider::class);`

In the file **App\Providers\AppServiceProvider** update the **register** method to add:

```php
/**
     * Register any application services.
     */
    public function register()
    {
        // ...
        $this->app->singleton('Illuminate\Contracts\Routing\ResponseFactory', function ($app) {
            return new \Illuminate\Routing\ResponseFactory
                $app['Illuminate\Contracts\View\Factory'],
                $app['Illuminate\Routing\Redirector']
            );
        });
        // ...
    }
```

**Comment by MagnificRogue** [comment]
Thanks @ismaail !

**Comment by andela-kakpobome** [comment]
Awesome sauce, thanks @ismaail . But note there's a missing opening `(` on the `return new \Illuminate\Routing\ResponseFactory` line, in case that gave someone issues.

**Comment by ismaail** [comment]
Thanks andela-kakpobome, typo fixed

**Comment by mstaack** [comment]
is there any information about this? this shouldn't be needed for a normal setup

@taylorotwell should we introduce this into the framework better?

**Comment by libasoles** [comment]
There's something missing there: 
new \Illuminate\**Contracts**\Routing\ResponseFactory

Anyway, this doesn't work for me (same stack: Lumen 5.4/PHP 7)

Maybe because there's nothing called 'Illuminate\Routing\Redirector'. Have you installed Laravel's Route component?

**Comment by pevawi** [comment]
I think this problem is not a Lumen problem but this is the same as this comment: [issue 504 comment from lsl](https://github.com/laravel/lumen-framework/issues/504#issuecomment-285936431)
To quote him:

> This seems to happen when laravel/framework gets required by something, composer wipes out lumens vendor/illuminate files with vendor/laravel/framework (laravel/framework composer.json says replace illuminate.) - this in turn includes illuminate/Foundation/helpers.php

Removing the offending package and removing the vendor directory and composer.lock file. Than running `composer update` solves the problem.

**Comment by AMS777** [comment]
As detailed by @pevawi, removing the vendor directory and running `composer update` works for me on Lumen 5.6 with following error:
```
Target [Illuminate\Contracts\Translation\Translator] is not instantiable.
```

**Comment by shahrukh4** [comment]
@libasoles Thanks brother you just saved me..

**Comment by driesvints** [comment]
This seems to be fixed in 5.7 so please upgrade. Thanks for reporting.

**Comment by kamleshwebtech** [comment]
Still getting 'Target is not instantiable' error

(1/1) BindingResolutionExceptionTarget [Illuminate\Contracts\Filesystem\Factory] is not instantiable.
--
in Container.php line 945
at Container->notInstantiable('Illuminate\\Contracts\\Filesystem\\Factory')in Container.php line 785
at Container->build('Illuminate\\Contracts\\Filesystem\\Factory')in Container.php line 658
at Container->resolve('Illuminate\\Contracts\\Filesystem\\Factory', array())in Container.php line 609
at Container->make('Illuminate\\Contracts\\Filesystem\\Factory', array())in Application.php line 260
